CHANGELOG.md

```diff
@@ -5,7 +5,7 @@ All notable changes to this project will be documented in this file.
 <a name="unreleased"></a>
 ## [Unreleased]
 
-
+- Added default value of 0 for metric filters
```

Comment:
changelog is auto-updating from commits, so no need to update it manually. I will fix.